src/contracts/listing.ts

```diff
@@ -15,4 +13,59 @@ export default class extends Deployed {
   at(w3:Web3, address:string, opts?:TransactOpts): Promise<boolean> {
     return super.at(w3, address, LISTING_ABI, opts)
   }
+
+  async isListed(hash:string, opts?:TransactOpts): Promise<Return> {
+    const deployed = this.requireDeployed()
+    let assigned = this.assignTransactOpts({gas: this.getGas('isListed')}, opts)
+    return [await deployed.methods.isListed(hash), assigned]
+  }
+
+  async withdrawFromListing(hash:string, amount:string, opts?:TransactOpts): Promise<Return> {
+    const deployed = this.requireDeployed()
+    let assigned = this.assignTransactOpts({gas: this.getGas('withdrawFromListing')}, opts)
+    return [await deployed.methods.withdrawFromListing(hash, amount), assigned]
+  }
+
+  async list(hash:string, opts?:TransactOpts): Promise<Return> {
+    const deployed = this.requireDeployed()
+    let assigned = this.assignTransactOpts({gas: this.getGas('list')}, opts)
+    return [await deployed.methods.list(hash), assigned]
+  }
+
+  async getListing(hash:string, opts?:TransactOpts): Promise<Return> {
+    const deployed = this.requireDeployed()
+    let assigned = this.assignTransactOpts({gas: this.getGas('getListing')}, opts)
+    return [await deployed.methods.getListing(hash), assigned]
+  }
+
+  // no longer exists in abi?
```

Comment:
you can grep the abi for it? don't think it needs to be in the HOC anyway, user's are never allowed to call it

Reply:
searching the abi returns no results.  mebbe the version in github is olde?